packages/templates/src/raw/indoor-garden-oasis/template.json

```diff
@@ -0,0 +1,3898 @@
+          "fontWeight": 700,
+          "borderRadius": {
+            "locked": true,
+            "topLeft": 2,
+            "topRight": 2,
+            "bottomRight": 2,
+            "bottomLeft": 2
+          },
+          "x": 70,
+          "y": 19,
+          "width": 294,
+          "height": 128,
+          "scale": 100,
+          "focalX": 50,
+          "focalY": 50,
+          "id": "b535f835-a8ac-4626-8874-33f8575feb6d",
+          "type": "text"
+        },
+        {
+          "opacity": 100,
+          "flip": { "vertical": false, "horizontal": false },
+          "rotationAngle": 0,
+          "lockAspectRatio": true,
+          "backgroundTextMode": "NONE",
+          "font": {
+            "family": "DM Sans",
+            "service": "fonts.google.com",
+            "fallbacks": ["sans-serif"],
+            "weights": [400, 500, 700],
+            "styles": ["regular", "italic"],
+            "variants": [
+              [0, 400],
+              [0, 500],
+              [0, 700],
+              [1, 400],
+              [1, 500],
+              [1, 700]
+            ],
+            "metrics": {
+              "upm": 1000,
+              "asc": 992,
+              "des": -310,
+              "tAsc": 992,
+              "tDes": -310,
+              "tLGap": 0,
+              "wAsc": 992,
+              "wDes": 310,
+              "xH": 496,
+              "capH": 700,
+              "yMin": -261,
+              "yMax": 967,
+              "hAsc": 992,
+              "hDes": -310,
+              "lGap": 0
+            }
+          },
+          "fontSize": 16,
+          "backgroundColor": { "color": { "r": 196, "g": 196, "b": 196 } },
+          "lineHeight": 1.4,
+          "textAlign": "initial",
+          "padding": { "locked": true, "hasHiddenPadding": false },
+          "content": "<span style=\"font-weight: 700; color: #235524\">Home gardening guide.</span>\n<span style=\"color: #235524\">Maria Contelo</span>",
+          "fontWeight": 700,
+          "borderRadius": {
+            "locked": true,
+            "topLeft": 2,
+            "topRight": 2,
+            "bottomRight": 2,
+            "bottomLeft": 2
+          },
+          "width": 180,
+          "height": 42,
+          "scale": 100,
+          "focalX": 50,
+          "focalY": 50,
+          "basedOn": "c788e51a-191f-4388-a6f9-e95b175deba8",
+          "id": "caa82a15-5242-43ff-8d6c-f8b16e6151ae",
+          "x": 70,
+          "y": 155,
+          "type": "text"
+        },
+        {
+          "opacity": 100,
+          "flip": { "vertical": false, "horizontal": false },
+          "rotationAngle": 0,
+          "lockAspectRatio": true,
+          "x": 214,
+          "y": 107,
+          "width": 147,
+          "height": 10,
+          "scale": 100,
+          "focalX": 50,
+          "focalY": 50,
+          "sticker": { "type": "homeGardenLine01" },
```

Comment:
@divyarajmasani This sticker does not exist.

Reply:
Updated the sticker name.